build/UIKit/lib/UIKitLib.vcxproj

```diff
@@ -110,6 +110,20 @@
     <ClangCompile Include="..\..\..\Frameworks\UIKit\UIView.mm" />
     <ClangCompile Include="..\..\..\Frameworks\UIKit\UIViewController.mm" />
     <ClangCompile Include="..\..\..\Frameworks\UIKit\UIWindow.mm" />
+    <ClangCompile Include="..\..\..\Frameworks\UIKit\WYPopoverController.m">
+      <ObjectiveCARC Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">true</ObjectiveCARC>
+      <ObjectiveCARC Condition="'$(Configuration)|$(Platform)'=='Debug|ARM'">true</ObjectiveCARC>
+      <ObjectiveCARC Condition="'$(Configuration)|$(Platform)'=='Release|ARM'">true</ObjectiveCARC>
+      <ObjectiveCARC Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">true</ObjectiveCARC>
+      <OptimizationLevel Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">Disabled</OptimizationLevel>
+      <OptimizationLevel Condition="'$(Configuration)|$(Platform)'=='Release|ARM'">Disabled</OptimizationLevel>
```

Comment:
That's pretty interesting. I know the codegen on the reference platform changed to memset structs to 0 for deterministic behaviour, and I know that we don't exactly support that.